tests/test_tremor_analysis.py

```diff
@@ -29,15 +29,15 @@ def test_2_extract_features_tremor_output(shared_datadir: Path):
 
 def test_3_tremor_detection_output(shared_datadir: Path):
```

Comment:
Waarom hebben je functies een numering? Is dat makkelijker om de stappen te volgen?